requirements.txt

```diff
@@ -0,0 +1,6 @@
+apache-beam[gcp]==2.34.0
+torch==1.11.0
+tokenizers==0.11.6
+transformers==4.17.0
+datasets==2.0.0
```

Comment:
any reason to pin `datasets==2.0.0` ? The latest version is `2.2.2`

Reply:
Nothing particular. We have been using 2.0.0 at Carted for some time. That's why. Can bump it up.